src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import './Calculator.css';
+
+function MyComponent() {
```

Comment:
- [ ] Calculator.js component should be class-based, you'll see in the next steps why this is important. 🐱
- [ ] At the same time, it looks like you're using `MyComponent` to declare the component. As you can see in your project, this works, but it could be confused if we have an application with 100 components, please, in place to use `MyComponent` as the component name, use `Calculator` to have better practices on your react application.